Wikipedia/Code/WMFImageController.swift

```diff
@@ -246,6 +246,14 @@ public class WMFImageController : NSObject {
         self.imageManager.imageCache.clearDisk()
     }
 
+    public func cacheImageData(imageData: NSData, url: NSURL){
```

Comment:
@coreyfloyd Could we name this "addImageDataToCache"? "cache" being both a noun and a verb and all.